lib/v2/focustaiwan/templates/article.art

```diff
@@ -0,0 +1 @@
+<img src="{{ $image }}"><div>{{ description }}</div>
```

Comment:
```suggestion
<img src="{{ $image }}"><div>{{@ description }}</div>
```
Use [Raw output](https://aui.github.io/art-template/docs/syntax.html#Raw-output) when you pass `.html()` into art-template. Otherwise, the HTML code will be escaped again by art-template which causes [middleware not working](https://github.com/DIYgod/RSSHub/blob/master/lib/middleware/parameter.js#L57-L116)(e.g., script not being removed and causing potential security problems).